executor/analyze.go

```diff
@@ -2489,6 +2526,66 @@ func FinishAnalyzeJob(ctx sessionctx.Context, job *statistics.AnalyzeJob, analyz
 	}
 }
 
+func prepareV2AnalyzeJobInfo(e *AnalyzeColumnsExec, retry bool) {
```

Comment:
it seems equal between introducing retry param for `describeV2AnalyzeJobInfo` and passing by `analyzeColumnsPushdownWithRetry`